src/OrbitGl/CallstackTooltipUtilsTest.cpp

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_join.h>
+#include <gtest/gtest.h>
+
+#include "CallstackTooltipUtils.h"
+#include "ClientData/ModuleAndFunctionLookup.h"
+
+using orbit_client_data::CallstackInfo;
+using orbit_client_data::CallstackType;
+using orbit_client_data::CaptureData;
+using orbit_client_data::LinuxAddressInfo;
+using orbit_client_data::ModuleManager;
+using orbit_grpc_protos::CaptureStarted;
+
+namespace orbit_gl {
+
+namespace {
+constexpr uint64_t kFrame1 = 0xADD5E55;
+constexpr uint64_t kOffsetInFunction = 0x12;
+constexpr const char* kModulePath = "/path/to/module";
+constexpr const char* kModuleName = "module";
+constexpr const char* kFunctionNameWithSpecialChars = "void foo<int>(const Foo&)";
+constexpr const char* kEscapedFunctionName = "void foo&lt;int&gt;(const Foo&amp;)";
+}  // namespace
+
+TEST(CallstackTooltipUtils,
+     FormatInnermostFrameOfEmptyCallstackForTooltipYieldsUnkownModuleAndFunctionName) {
+  CallstackInfo empty_callstack{{}, CallstackType::kDwarfUnwindingError};
+  CaptureData capture_data{{}, {}, {}, CaptureData::DataSource::kLiveCapture};
+  ModuleManager module_manager{};
+  FormattedModuleAndFunctionName module_and_function_name =
+      FormatInnermostFrameOfCallstackForTooltip(empty_callstack, capture_data, module_manager);
+
+  EXPECT_EQ(module_and_function_name.module_name,
+            absl::StrCat("<i>", orbit_client_data::kUnknownFunctionOrModuleName, "</i>"));
+  EXPECT_EQ(module_and_function_name.function_name,
+            absl::StrCat("<i>", orbit_client_data::kUnknownFunctionOrModuleName, "</i>"));
+}
+
+TEST(CallstackTooltipUtils, FormatInnermostFrameOfCallstackForTooltipPerformsHTMLEscaping) {
```

Comment:
```suggestion
TEST(CallstackTooltipUtils, FormatInnermostFrameOfCallstackForTooltipPerformsHtmlEscaping) {
```
Style guide: "prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC()."

Reply:
Done.